clients/src/main/java/org/apache/kafka/clients/admin/OffsetSpec.java

```diff
@@ -73,18 +73,20 @@ public static OffsetSpec maxTimestamp() {
     }
 
     /**
-     * Used to retrieve the offset with the local log start offset,
-     * log start offset is the offset of a log above which reads are guaranteed to be served
-     * from the disk of the leader broker, when Tiered Storage is not enabled, it behaves the same
-     * as the earliest timestamp
+     * Used to retrieve the local log start offset.
+     * Local log start offset is the offset of a log above which reads
+     * are guaranteed to be served from the disk of the leader broker.
+     * <br/>
+     * Note: When tiered Storage is not enabled, it behaves the same as retrieving the earliest timestamp offset.
      */
     public static OffsetSpec earliestLocalSpec() {
```

Comment:
This is unrelated to this PR. The other specs don't use `Spec` as postfix. Maybe we should align the naming before release?

Reply:
OK, I'll remove the postfix